### PR TITLE
workflow: don't send updates to legacy and approve them on labs

### PR DESCRIPTION
**Pull request by ammirate**
Signed-off-by: Antonio Cesarano <cesarano2607@gmail.com>

This PR removes the window time in which articles are automatically rejected since we can recognize them now as updates, using the matcher.
Such updates are now being sent back to legacy, potentially overwriting human work.
Thus, updates in this PR are not sent to legacy at all.

## Related Issue
https://github.com/inspirehep/inspire-next/issues/3124

## Checklist:

- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [x] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/integration/workflows/test_arxiv_workflow.py [hunk 1]
```diff
@@ -373,3 +375,46 @@ def test_match_in_holdingpen_different_sources_continues(
...
+)
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.beard.json_api_request',
+    side_effect=fake_beard_api_request,
+)
+@mock.patch(
+    'inspirehep.modules.workflows.tasks.magpie.json_api_request',
+    side_effect=fake_magpie_api_request,
+)
+def test_arxiv_update_overwrites_record_on_labs_but_not_in_legacy(
```
Comment (michamos) [inline tests/integration/workflows/test_arxiv_workflow.py]: why should it overwrite the record on labs? don't we want to maintain it in sync with legacy?

Comment (ammirate) [reply tests/integration/workflows/test_arxiv_workflow.py]: For now yes, and in fact, after the robot upload the record comes back from legacy and overwrites whatever there is.

Comment (michamos) [reply tests/integration/workflows/test_arxiv_workflow.py]: but if it's an update, there is no robotupload, so the record might never come back.

Comment (ammirate) [reply tests/integration/workflows/test_arxiv_workflow.py]: That's true, you are right. So, let's throw them away.

**Comment by ammirate** [comment]
ci test please

**Pull request merged**